### PR TITLE
Correct grammar errors

### LINKED ISSUE #151: Fix the grammar for the report
We overlooked too many grammar and spelling mistakes. For example, in the use case description for "Assign Tasks", there is an "and and", and there are many other verbs that are not conjugated.

I suggest using some tool for checking the grammar and spelling, such as [LanguageTool](https://languagetool.org/).

### PR DESCRIPTION
**Pull request by Huy-Ngo**
I'm not sure if this resolves #151, but it does correct a lot of mistakes and improves a bit writing style.

### CODE REVIEW

File: docs/source/req/use-case/collab.rst [hunk 1]
```diff
@@ -313,15 +273,12 @@ Basic Flow
 Alternative Flow
 """"""""""""""""
 
-Thread existed
-   If in step 5, an user recognizes that another thread has the solution,
-   he/she could mark the thread as existed. The system require that user
-   to provide clear instruction leading to the existed thread.
+None.
```
Comment (McSinyx) [inline docs/source/req/use-case/collab.rst:276]: Let's remove this `else` and unnest this then.

**Review by McSinyx — COMMENTED** [review]
I'd argue over the form of the verb `log` used in various places but I guess language involves as time goes by.

**Comment by Huy-Ngo** [comment]
> I guess language involves as time goes by.

you mean "evolves"?

**Review by McSinyx — APPROVED** [review]
Yea LOL

**Pull request merged**